app/scripts/modules/core/authentication/authentication.initializer.service.ts

```diff
@@ -5,11 +5,8 @@ import {IModalService, IModalStackService} from 'angular-ui-bootstrap';
 
 import {IDeckRootScope} from '../domain/deckRootScope';
 import {REDIRECT_SERVICE, RedirectService} from './redirect.service';
-import {AUTHENTICATION_SERVICE, AuthenticationService} from './authentication.service';
+import {AUTHENTICATION_SERVICE, AuthenticationService, IAuthResponse} from './authentication.service';
 
-interface IAuthResponse {
```

Comment:
Agree with @danielpeach here.  Unless the interface is used application-wide, it should go in the same file where it's primarily used.